tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf8 -*-
+"""
+Test cases in this file are demonstrating usage of workload storage utilization
+fixtures, and are not expected to be executed in any real test run (hence
+all tests are marked with ``libtest`` marker).
+
+You can execute test cases here to run the workflow implemented in the
+fixtures. Assuming that you are in root directory of ``ocs-ci`` repository and
+that your environment is fully configured, you can do this for example like
+this:
+
+.. code-block:: console
+
+    (venv) [ocsqe@localhost ocs-ci]$ run-ci --cluster-path ~/data/cluster tests/manage/monitoring/test_workload_fixture.py -vvv --pdb
```

Comment:
In [usage](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/usage.md#deployment-of-cluster) documentation (and on other places) we are using `/home/my_user/my-ocs-dir` as *cluster dir*, so I would suggest to use it here in the similar way.

Reply:
ok